### DISCUSSION TITLE
Can an Image Field Be Added To The Media Preview

### DISCUSSION THREAD

**Discussion by markslloyd93** [PiranhaCMS/piranha.core#1979] [Q&A] [answered]
Hi,

Is it currently possible to add an image field to the media preview modal? I'm currently trying to add the option where a CMS user can upload a JPG or PNG & when clicking into the media preview have an option to upload a webp version. 

The only information i have found so far is for adding new meta data properties but it looks like only string fields can be added.

[Adding Meta Data Properties](https://piranhacms.org/docs/master/content/media)

**Comment by tidyui** [comment, marked as answer]
Hi there! Media-versions are only intended to be scaled/resized/cropped versions of the existing image that have been processed by Piranha internally. There's no support for uploading a completely different image and attach it to an image/asset.
